introduction.md

```diff
@@ -1,4 +1,7 @@
 # Introduction #
 this is my intro
 
-Favorite path is JavaScript
\ No newline at end of file
+Favorite path is JavaScript
+
+* Git
+* JavaScript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?